package.json

```diff
@@ -293,6 +293,7 @@
     "enzyme-adapter-react-16": "1.15.6",
     "ethereum-blockies": "github:brave/blockies#07ebdfa3ed79fceec8234ef8c880ebdefd79bf0a",
     "ethereumjs-util": "7.0.9",
+    "trezor-connect": "8.2.1-extended",
```

Comment:
this needs to be sorted to match original

Reply:
done